File: src/main/java/com/likelionknu/applyserver/discord/data/exception/DiscordNotificationException.java
package com.likelionknu.applyserver.discord.data.exception;

import com.likelionknu.applyserver.common.response.ErrorCode;
import com.likelionknu.applyserver.common.response.GlobalException;

public class DiscordNotificationException extends GlobalException {
    public DiscordNotificationException() {
        super(ErrorCode.INTERNAL_SERVER_ERROR);
    }
}

File: src/main/kotlin/com/likelionknu/applyserver/discord/data/exception/DiscordNotificationException.kt
package com.likelionknu.applyserver.discord.data.exception

import com.likelionknu.applyserver.common.response.ErrorCode
import com.likelionknu.applyserver.common.response.GlobalException

class DiscordNotificationException : GlobalException(ErrorCode.INTERNAL_SERVER_ERROR)